app/views/posts/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>
+This shows the details of apost with id = #{post.find(:id)}
+written by an author = #{users.find(:id)}
+</h1>
```

Comment:
- [ ] As per the requirement, this page should seem like this. Please fix it. 
![image](https://github.com/obibaadoma/Blog_App/assets/91486791/d3ce1fb7-2f24-4482-a304-1efce6ee5695)

Reply:
done